gateway/tests/integration_tests/setup.rs

```diff
@@ -92,7 +92,7 @@ pub async fn test_setup_with_config(
     test_name: &str,
     sp_port: SpPort,
     mut server_config: omicron_gateway::Config,
-    sp_sim_config: &mut sp_sim::Config,
```

Comment:
Accidentally put this change in, though I think it makes sense.

Reply:
Agree it makes sense, as long as tests still pass, which they should if it builds.